.github/workflows/codespell.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Codespell
```

Comment:
Should add a comment somewhere near the top of this workflow file to explain why it gets its own workflow file (so that it runs on the docs as well as the code)